src/components/UserForm/EmailStatusMessage.test.js

```diff
@@ -0,0 +1,87 @@
+import { render, screen, waitFor } from '@testing-library/react'
+import React from 'react'
+import EmailStatusMessage from './EmailStatusMessage.js'
+
+jest.mock('@dhis2/ui', () => ({
+    ...jest.requireActual('@dhis2/ui'),
+    colors: {
+        green600: 'green',
+        red600: 'red',
+        default: 'black',
+    },
+    IconCheckmarkCircle16: () => <div data-test="icon-checkmark-circle" />,
+    IconWarning16: () => <div data-test="icon-warning" />,
+    IconInfo16: () => <div data-test="icon-info" />,
+}))
+
+describe('EmailStatusMessage', () => {
```

Comment:
maybe add a test condition where `<EmailStatusMessage emailVerified={true} enforceVerifiedEmail={false} />` and then you will cover the possible scenarios here?